test/src/locator.spec.ts

```diff
@@ -113,6 +113,9 @@ describe('Locator', function () {
         .on(LocatorEmittedEvents.Action, () => {
           clicked = true;
         })
+        .expect((element): element is HTMLElement => {
```

Comment:
IMO it should be possible to click without a need for expect: requiring the expect is too verbose. It should just throw an error if the element is not clickable instead of requiring boilerplate because the outcome is the same, it is going to fail at runtime.